tests/python/unittest/test_gluon_estimator.py

```diff
@@ -260,7 +251,8 @@ def test_context():
                     loss=loss,
                     metrics=metrics)
     # input list of context
-    ctx = [mx.gpu(0), mx.gpu(1)]
+    ctx = [mx.cpu() for _ in range(2)]
```

Comment:
change this ctx=[ list of gpu ctxs] if num_gpus >1 else [mx.cpu()]

Reply:
Done. Thanks!